hubblestack/extmods/hubble_mods/pkg.py

```diff
@@ -0,0 +1,129 @@
+# -*- encoding: utf-8 -*-
+"""
+Module for running pkg command. Same can be used in both Audit/FDG
+
+Audit Example:
+---------------
+check_unique_id:
+  description: 'pkg check'
+  tag: 'ADOBE-01'
+  implementations:
+    - filter:
+        grains: 'G@osfinger:CentOS*Linux-7'
+      hubble_version: '>3 AND <7 AND <8'
+      module: pkg
+      items:
+        - args:
+            name: perl*
+
+FDG Example:
+------------
+main:
+  description: 'pkg check'
+  module: pkg
+  args:
+    name: perl*
+Mandatory parameters:
+    name - name of package
+Multiple package names can be provided in a single implementation under attribute: "items"
+
+Note: Comparison logic is moved to comparators. Module will just invoke the pkg command.
+Comparator compatible with this module - dict
+
+Sample Output:
+{
+    'perl-srpm-macros': '1-8.el7',
+    'perl-parent': '1:0.225-244.el7',
+    'perl-podlators': '2.5.1-3.el7',
+    'perl-Pod-Escapes': '1:1.04-295.el7',
+    'perl-Encode': '2.51-7.el7'
+}
+"""
+
+import logging
+import fnmatch
+
+import hubblestack.extmods.module_runner.runner_utils as runner_utils
+from hubblestack.utils.hubble_error import HubbleCheckValidationError
+
+log = logging.getLogger(__name__)
+
+
+def validate_params(block_id, block_dict, chain_args=None):
+    """
+    Validate all mandatory params required for this module
+
+    :param block_id:
+        id of the block
+    :param block_dict:
+        parameter for this module
+    :param chain_args:
+        Chained argument dictionary, (If any)
+        Example: {'result': "/some/path/file.txt", 'status': True}
```

Comment:
it should show a package name, instead of a filepath

Reply:
done